helix-term/src/commands.rs

```diff
@@ -5701,27 +5701,29 @@ async fn shell_impl_async(
         process.wait_with_output().await?
     };
 
-    if !output.status.success() {
-        if !output.stderr.is_empty() {
-            let err = String::from_utf8_lossy(&output.stderr).to_string();
```

Comment:
Why not use the lossy function like we did before? That way at least something is displayed

Reply:
The thought was to treat `stderr` as how the `stdout` was before, which was handled this way. I can go back to lossy though.